src/commands/Birthdays/setBirthday.ts

```diff
@@ -0,0 +1,144 @@
+import {Args, PieceContext} from '@sapphire/framework';
+import {Message, MessageActionRow, MessageButton} from 'discord.js';
+import {getSettings} from '../../database/models/SettingsModel';
+import {BediEmbed} from '../../lib/BediEmbed';
+import colors from '../../utils/colorUtil';
+import {surroundStringWithBackTick} from '../../utils/discordUtil';
+import {updateBirthday} from '../../database/models/BirthdayModel';
+
+const {Command} = require('@sapphire/framework');
+
+const YEAR_TO_SAVE = 2025;
+
+module.exports = class SetBirthdayCommand extends Command {
+  constructor(context: PieceContext) {
+    super(context, {
+      name: 'setbirthday',
+      aliases: ['sb'],
+      description: 'Sets the users birthday.',
+      //TODO: Add disclaimer that anyone in a BediBot server can find your birthday (month and day) - put in detailed description
```

Comment:
None of the commands so far have a detailed description, it's going to be used for the help command. I have an issue to do them all at once since they will probably be similar.

Reply:
Also I want to wait for when I do the help command since the format of these descriptions will probably be dictated by how we want the help command to look. If I write it now there's a good change I end up changing it anyways. I just wrote that note so that I remember to include that information later.